delta/app/src/test/resources/supervision/supervision-running-proj-response.json

```diff
@@ -0,0 +1,43 @@
+{
+  "@context" : "https://bluebrain.github.io/nexus/contexts/supervision.json",
```

Comment:
It looks like it requires a reformating from IntelliJ (there should not be a space between the end of `context` and the colon)